test/unit/package_test.rb

```diff
@@ -2,403 +2,145 @@
 
 class PackageTest < ActiveSupport::TestCase
   setup do
-    @weight = 100
-    @dimensions = [5, 6, 7]
+    @weight = Measured::Weight(500, :grams)
+    @length = Measured::Length(30, :cm)
+    @width = Measured::Length(25, :cm)
+    @height = Measured::Length(15, :cm)
```

Comment:
Switch one of these to another unit. That would have failed the `.value` bug I pointed out above.